docs/curl.md

```diff
@@ -0,0 +1,26 @@
+# The cURL Command
+
+## Summary
+The `cURL` command stands for "client URL." Essentually, the command permits the transfer of data to and from a server, with specifying the location of the data that you desire to send through the form of a URL. It is especially helpful in sending API requests, checking that servers are recieving requests.
+
+## Basic Command Structure
+For all descriptions below, the dollar sign indicates that BASH command prompt.
+
+$ `cURL [options] [url]`
```

Comment:
I think it's worth noting that the user can also type the command as `curl` and it works the same.